.circleci/config.yml

```diff
@@ -2,7 +2,13 @@ version: 2.1
 
 # heavily inspired by https://raw.githubusercontent.com/pinax/pinax-wiki/6bd2a99ab6f702e300d708532a6d1d9aa638b9f8/.circleci/config.yml
 
-common_go_v1_21_3: &common_go_v1_21_3
+parameters:
+  go_version:
+    # change the common_go_vX_Y_Z when you change this default value
+    default: "1.22.3"
+    type: string
+
+common_go_v1_22_3: &common_go_v1_22_3
```

Comment:
Could we just name this `common_go` and avoid having to update each job when we update go versions in the future?

Reply:
hah yes... that makes too much sense